helix-term/src/config.rs

```diff
@@ -20,6 +21,25 @@ pub struct LspConfig {
     pub display_messages: bool,
 }
 
+impl Config {
+    pub fn load(config_path: PathBuf) -> Result<Config, Error> {
+        match std::fs::read_to_string(config_path) {
+            Ok(config) => Result::Ok(toml::from_str(&config)
+                .map(merge_keys)
+                .unwrap_or_else(|err| {
+                    eprintln!("Bad config: {}", err);
+                    eprintln!("Press <ENTER> to continue with default config");
+                    use std::io::Read;
+                    // This waits for an enter press.
+                    let _ = std::io::stdin().read(&mut []);
+                    Config::default()
+                })),
```

Comment:
Hmm, it seems that this case (and probably NotFound too) should be handled outside this method. This print + blocking behavior is only intended to happen during `hx` startup. You should be able to catch the error and use `editor.set_error()` when reloading